Source/JavaScriptCore/assembler/ARM64Assembler.h

```diff
@@ -3297,7 +3299,7 @@ class ARM64Assembler {
     template<int datasize>
     ALWAYS_INLINE void stur(FPRegisterID rt, RegisterID rn, int simm)
     {
-        CHECK_DATASIZE_SIMD();
+        CHECK_FP_MEMOP_DATASIZE();
```

Comment:
Does this work with data size == 8? `CHECK_FP_MEMOP_DATASIZE()` seems to accept that?

Reply:
Yes, it works for 8bit too.